addons/mail/static/src/js/chatter.js

```diff
@@ -111,6 +112,16 @@ var Chatter = Widget.extend({
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * resets _suggestedPartnersProm and call _openComposer
+     * if composer is already open, to update suggested recipients
+     */
+    resetSuggestedPartners() {
+        this._resetSuggestedPartners();
+        if (this._isComposerOpen && !this._composer.options.isLog) {
+            this._openComposerMessage();
+        }
```

Comment:
Only suggested recipients on composer should be re-render.
Open/close is not a good idea, because it clears content that was in the composer.
Also `chatter_composer` should decide re-render based on `isLog`, not `chatter`.

So prefer calling a public method on the composer to re-render only the suggested recipients part, like `.updateSuggestedPartners()`?